packages/kubernetes-1.29/prestart-pull-pause-ctr.conf

```diff
@@ -6,4 +6,5 @@ ExecStartPre=/usr/bin/host-ctr \
     pull-image \
     --source=${POD_INFRA_CONTAINER_IMAGE} \
     --registry-config=/etc/host-containers/host-ctr.toml \
-    --skip-if-image-exists=true
+    --skip-if-image-exists=true \
+    --labels="io.cri-containerd.pinned=pinned"
```

Comment:
Do we need a migration for this?

Reply:
No, since the PR isn't changing the API shape nor the API store values.

Reply:
I realized this after I hit the wrong button and submitted the comment out of band. But left it in case I was wrong on the second guess.